src/irc/ConnectionInstance.ts

```diff
@@ -42,7 +43,8 @@ const FLOOD_PROTECTION_DELAY_MS = 700;
 const THROTTLE_WAIT_MS = 20 * 1000;
 
 // String reply of any CTCP Version requests
-const CTCP_VERSION = 'matrix-appservice-irc, part of the Matrix.org Network';
+const CTCP_VERSION = 
+    (homeserverName: string) => `matrix-appservice-irc ${getBridgeVersion()} bridged via ${homeserverName}`;
```

Comment:
Do we need to escape these strings do you think?

Reply:
I don't think this can break anything on accident, and if someone wants to break it deliberately then there are easier options for someone controlling the HS and the bridge :)